scripts/deployment/deploy.sh

```diff
@@ -245,15 +285,16 @@ deploy_openlibrary() {
     echo "Pull the latest docker images..."
     # We need to fetch by the exact image sha, since the registry mirror on the prod servers
     # has a cache which means fetching the `latest` image could be stale.
-    OLBASE_SHA=$(echo $IMAGE_META | jq -r '.images[0].digest')
+    OLBASE_DIGEST=$(echo $IMAGE_META | jq -r '.images[0].digest')
     for SERVER in $SERVERS; do
-        echo -n "   $SERVER ... "
+        echo "   $SERVER ... "
         ssh -t $SERVER "
-            docker pull openlibrary/olbase@$OLBASE_SHA
-            echo 'FROM openlibrary/olbase@$OLBASE_SHA' | docker build --tag openlibrary/olbase:latest -f - .
+            set -e;
+            docker pull openlibrary/olbase@$OLBASE_DIGEST
+            echo 'FROM openlibrary/olbase@$OLBASE_DIGEST' | docker build --tag openlibrary/olbase:latest -f - .
             COMPOSE_FILE='$COMPOSE_FILE' HOSTNAME='\$HOSTNAME' docker compose --profile $SERVER pull
```

Comment:
Fix host machine hostname being used during restart.
```suggestion
            COMPOSE_FILE='$COMPOSE_FILE' HOSTNAME=\$HOSTNAME docker compose --profile $SERVER pull
```